src/main/scala/org/broadinstitute/dsde/firecloud/webservice/UserApiService.scala

```diff
@@ -71,7 +65,9 @@ trait UserApiService extends HttpService with PerRequestCreator with FireCloudRe
 
   lazy val log = LoggerFactory.getLogger(getClass)
 
-  val routes =
+  val trialServiceConstructor: () => TrialService
```

Comment:
refactoring UserApiService was necessary to get the `trialServiceConstructor` properly injected from `FireCloudServiceActor` ... unfortunately this meant I had to rename `routes` to `userServiceRoutes` and that cascaded changes elsewhere.